core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/Tenant.java

```diff
@@ -32,8 +35,10 @@
 
     private String id;
 
+    @JsonInclude(Include.NON_EMPTY)
     private List<AggregationTemplate> aggregationTemplates = new ArrayList<>();
```

Comment:
I can see only way to do this: maintain a separate model, dedicated to transfer in the JAX-RS and WebSocket API. In case you're worried about bean mappings, Gunnar (from the Hibernate team) has created a nice tool for this: http://mapstruct.org/